browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -77,7 +77,7 @@ void EphemeralStorageTabHelper::ReadyToCommitNavigation(
   if (new_domain == previous_domain)
     return;
 
-  CreateEphemeralStorageAreasForDomainAndURL(new_domain, new_url);
+  CreateEphemeralStorageAreasForDomainAndURL(std::move(new_domain), new_url);
```

Comment:
these changes don't appear to be related to this PR and also it should be using `const&` like the others mentioned here

Reply:
it's an unnecessary copy, noticed it while was working on ```TLDEphemeralLifetime```, fixed on the way.

Reply:
replaced first arg with ```const&``` in ```CreateEphemeralStorageAreasForDomainAndURL```